ability.go

```diff
@@ -7,8 +7,8 @@ type ListAbilityResponse struct {
 	Abilities []string `json:"abilities"`
 }
 
-// ListAbilities lists all abilities on your account. It's recommended to use
-// ListAbilitiesWithContext instead.
+// ListAbilities lists all abilities on your account.
+// Deprecated: Use ListAbilitiesWithContext instead.
```

Comment:
@dsymonds I took a look at the docs on this comment, and it seems like it expects the deprecation notice to be on its own paragraph: https://github.com/golang/go/wiki/Deprecated

> To signal that an identifier should not be used, add a paragraph to its doc comment that begins with Deprecated: followed by some information about the deprecation, and a recommendation on what to use instead, if applicable.
> 
> The paragraph does not have to be the last paragraph in the doc comment.

Would you be able to update the change to look more like:

```go
// ListAbilities lists all abilities on your account.
//
// Deprecated: Use ListAbilitiesWithContext instead.
```

Reply:
Sure, done.